repositories/dbmodels/db_api_key.go

```diff
@@ -10,7 +10,7 @@ import (
 type DBApiKey struct {
 	Id             string             `db:"id"`
 	OrganizationId string             `db:"org_id"`
-	Key            string             `db:"key"`
+	Hash           string             `db:"key"`
```

Comment:
We may want to alter the table column but I think this should be `key` until we implement hashing